.github/workflows/build-and-publish-release-artifacts.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Publish Release Artifacts
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ^1.17.0
```

Comment:
I think we can exclude this so that it always builds with latest (we probably want to run 1.18+ right now anyway).

Reply:
Fixed.